affiliate/src/productCard/ProductCard.jsx

```diff
@@ -18,26 +18,44 @@ const ProductCard = forwardRef(({
   textColor,
   linkColor,
   backgroundColor,
+  isAmp,
+  publicDomain,
 }, ref) => {
   if (!productGuid) {
     return null;
   }
-  const options = serializeOptions({
-    displayDescription,
-    displayImage,
-    cardRadius,
-    cardShadow,
-    textColor: convertColorValue(textColor),
-    linkColor: convertColorValue(linkColor),
-    backgroundColor: convertColorValue(backgroundColor),
-  });
+  const options = serializeOptions(
+    isAmp ? '&' : ',',
+    {
+      displayDescription,
+      displayImage,
+      cardRadius,
+      cardShadow,
+      textColor: convertColorValue(textColor),
+      linkColor: convertColorValue(linkColor),
+      backgroundColor: convertColorValue(backgroundColor),
+    },
+  );
+  if (!isAmp) {
+    return (
+      <div
+        ref={ref}
+        data-organic-affiliate-integration="product-card"
+        data-organic-affiliate-integration-options={options}
+        data-organic-affiliate-product-guid={productGuid}
+      />
+    );
+  }
+  const src = `${publicDomain}/integrations/affiliate/product-card?guid=${productGuid}&${options}`;
```

Comment:
Shouldn't we translate div into amp-iframe in the Affiliate SDK instead?

Reply:
Can we? I was under the impression that the Affiliate SDK wouldn't be able to run on AMP pages (at least yet) since arbitrary JavaScript isn't allowed, and it doesn't look like our SDK code is embedded using `amp-script`. If we can, then I am all for doing it via the SDK.

Reply:
> I was under the impression that the Affiliate SDK wouldn't be able to run on AMP pages

Probably you're right. I don't know what AMP is. We translate `div` into `iframe` by Affiliate SDK and thought that doing so would be consistent. If it isn't possible technically, then it's ok

Reply:
I discussed how AMP works in the wordpress plugin with @sergunich . The same page could be rendered for AMP and not. For instance, for desktop browsers a page is rendered in the regular mode and in AMP mode for mobile devices. We should determine how to render the product card in runtime. We can do this on backend side.

In the PageInjector class we have amp-dependent logic
https://github.com/orgnc/wordpress-plugin/blob/master/Organic/PageInjection.php#L28-L51

What do you think if we'll insert an `amp-iframe` inside the `div[data-organic-affiliate-integration="product-card"]` elements there? I think `injectConnatix` is a good example of how it could be done.
https://github.com/orgnc/wordpress-plugin/blob/50ce425e6e0a30461371e9bdf4e0c578f9fc4beb/Organic/AmpAdsInjector.php#L65-L93

Reply:
Ah, ok, this makes sense. I will switch to this. I agree that this is better.